weblogic-azure-aks/src/main/resources/aks_tooling_well_tested_versions.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
To minimize the impact, this PR add a separated file. I've filed issue #309  to refactor the configurations.